### PR TITLE
[documentation] Remove duplicate section in Messaging > Consumers > Acknowledgement

### PR DESCRIPTION
**Pull request by yehancha**
### Motivation

In [Messaging > Consumers > Acknowledgement](https://pulsar.apache.org/docs/en/2.7.1/concepts-messaging/#acknowledgement) section in the documentation, it is described the number of ways the messages get acknowledged. This description was duplicated; once in a paragraph form and then in a point form. One should be removed to keep the documentation flow.

### Modifications

The description in the paragraph form is removed since the point form is more clearer.

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@yehancha good catch!

Could you please help update all affected doc versions? Thanks

**Comment by yehan-macrometa** [comment]
> @yehancha good catch!
> 
> Could you please help update all affected doc versions? Thanks

@Anonymitaet Thanks! Will do.

**Comment by yehancha** [comment]
> @yehancha good catch!
> 
> Could you please help update all affected doc versions? Thanks

@Anonymitaet Thanks! Will do.

**Comment by yehancha** [comment]
@Anonymitaet Fixed in all versions.

**Review by Anonymitaet — APPROVED** [review]

**Comment by Anonymitaet** [comment]
@yehancha thanks! You can ping me once all tests pass, then I can merge this PR.

**Comment by yehancha** [comment]
@Anonymitaet Noted with thanks! I think everything has passed.

**Comment by yehancha** [comment]
@Anonymitaet - Are we waiting for @Huanli-Meng's review?

**Review by Huanli-Meng — APPROVED** [review]

**Pull request merged**
